components/Ink/src/private/MDCSimpleInkLayer.m

```diff
@@ -0,0 +1,156 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCSimpleInkLayer.h"
+#import "MaterialMath.h"
+
+static NSString *const kMDCSimpleInkLayerOpacityString = @"opacity";
+static NSString *const kMDCSimpleInkLayerPositionString = @"position";
+static NSString *const kMDCSimpleInkLayerScaleString = @"transform.scale";
+
+@implementation MDCSimpleInkLayer
+
+- (instancetype)initWithLayer:(id)layer {
```

Comment:
initWithLayer: is used as the copy constructor.
This method should be the standard init where we set the defaults.
initWithLayer should check if the layer parameter is an MDCSimpleInkLayer and if so, copy its values into the newly created layer.